modules/microshift-install-bootc-creating-vm.adoc

```diff
@@ -22,15 +22,21 @@ You can create a virtual machine by using the {op-system-base-full} boot ISO ima
 
 . Copy the downloaded file to the `/var/lib/libvirt/images` directory.
 
-. Use your values to configure the following environment variables:
+. Configure the VMNAME environment variable with your value by running the following command:
 +
 [source,terminal,subs="attributes+"]
 ----
-VMNAME=microshift-{product-version}-bootc
-NETNAME=default
+$ VMNAME=microshift-{product-version}-bootc
 ----
 
-. Create a {op-system-base-full} virtual machine with 2 cores, 2GB of RAM and 20GB of storage by running the following command:
+. Configure the NETNAME environment variable with your value by running the following command:
++
+[source,terminal]
```

Comment:
```suggestion
[source,terminal,subs="attributes+"]
```

Reply:
Should "default" be an attribute instead?